_docs/introduction/quick_start.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; url=http://localhost:4000/docs/install/quickstart" />
```

Comment:
I did - there's a few links that still point to the this page, so Chris asked for a redirect to the right quickstart.